### PR TITLE
Update and improve the API landing page, add more topics

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

Starting a multi-step process to build out and improve the API docs

### CODE REVIEW

File: docs/tools/api.rst [hunk 1]
```diff
@@ -1,48 +1,44 @@
-Aiven API overview
-==================
+Aiven API
+=========
 
-The Aiven API allows you to perform any tasks on Aiven from a client of your choice.
+Use the Aiven API to programmatically perform any task that you can do through the web interface. This is an ideal way to automate tasks involving Aiven at every stage of your workflow.
```
Comment (Andrew-Hindle) [inline docs/tools/api.rst:4]:
```suggestion
Use the Aiven API to perform any programming task that you can perform through the web interface. This is an ideal way to automate tasks, involving Aiven at every stage of your workflow.
```

File: docs/tools/api.rst [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
 
-    sequenceDiagram
-        participant Client
-        participant Aiven
+Authentication
+--------------
 
-        Client->>Aiven: Request with access token
-        Aiven->>Client: Response {JSON}
+Some endpoints (but not quite all) require authentication. You will need an authentication token from the `profile section of your Aiven console <https://console.aiven.io/profile/auth>`_.
```
Comment (Andrew-Hindle) [inline docs/tools/api.rst:29]:
```suggestion
Some (but not quite all) endpoints require authentication. You'll need an authentication token from the `profile section of your Aiven console <https://console.aiven.io/profile/auth>`_.
```

**Review by Andrew-Hindle — COMMENTED** [review]
Only a couple of minor edits suggested here.

File: docs/tools/api.rst (hunk 1 above)
Comment (Andrew-Hindle) [reply docs/tools/api.rst:4]: That just repeats the original text, but it clears up what "programmatically" means in this context :-D it's fine the way it was, then.

File: docs/tools/api.rst [hunk 3]
```diff
@@ -1,48 +1,44 @@
...
+
+* Use with CI (Continuous Integration) to spin up data platforms on Aiven for use during test runs.
+
+* Integrate with other parts of your existing automation setup to achieve complex tasks.
 
-* Try `Aiven on Postman <https://www.postman.com/aiven-apis/workspace/aiven/documentation/21112408-1f6306ef-982e-49f8-bdae-4d9fdadbd6cd>`_ and make your first calls to our API.
+* Deploy and tear down development or demo platforms on a schedule.
 
-* You will need an authentication token from the `profile section of your Aiven console <https://console.aiven.io/profile/auth>`_.
+We make the API available to all Aiven users. It is also the engine behind our web console, so you should find that all operations are also available via API.
```
Comment (Andrew-Hindle) [inline docs/tools/api.rst:14]:
```suggestion
We make the API available to all Aiven users. It is also the engine behind the Aiven Console, so you should find that all operations are also available through the API.
```

Comment (Andrew-Hindle) [reply docs/tools/api.rst:14]: Assuming this is the Aiven Console you're talking about, Slab says we need to word it like this.

File: docs/tools/api/concepts/tokens.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
+Work with API authorization tokens
+==================================
+
+The Aiven API uses tokens for authentication. In this article, we will cover some of the key considerations for working with API tokens.
```
Comment (Andrew-Hindle) [inline docs/tools/api/concepts/tokens.rst:4]:
```suggestion
The Aiven API uses tokens for authentication. In this article, we cover some of the key considerations when working with API tokens.
```

File: docs/tools/api/concepts/tokens.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Limit token expiry
+------------------
+
+Think about how long the token needs to be valid for; this will vary with the situation that the token is used for.
```
Comment (Andrew-Hindle) [inline docs/tools/api/concepts/tokens.rst:9]:
```suggestion
Think about how long the token needs to be valid for; this varies depending on what the token is used for.
```

**Review by Andrew-Hindle — COMMENTED** [review]
Generally fine, some minor comments / product naming

**Comment by lornajane** [comment]
Sorry to tag a bunch of you, Andy had a look at the content but it could do with a documentation maintainer to review as well? Merge if you're happy!

**Comment by dewan-ahmed** [comment]
Hi, I'm taking a 👀

**Review by dewan-ahmed — COMMENTED** [review]
Hi @lornajane this is great. I updated few words via https://github.com/aiven/devportal/pull/1376/commits/5446bc5f87db0327c92b072d77924e184aa79eb0 (for example, one sentence was in first-person).

Considering that we have **https://docs.aiven.io/docs/platform/concepts/authentication-tokens.html**, I propose **docs/tools/api/concepts/tokens.html** to either be included in that article or add under **docs/platform/concepts** as "authentication token best practices". 

wdyt?

**Comment by lornajane** [comment]
I think: Great suggestion! Changes made.

**Pull request merged**